app.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra/base'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  get '/' do
+    @peeps = Peep.all
+    erb :chitter_feed
+  end
+
+  post '/' do
+    p ENV
+    Peep.post(text: params['text'])
+    redirect '/'
+  end
+
+  run! if app_file == $0
```

Comment:
Can you remind me what this line does? I see it everywhere but haven't included it in mine